frontend/public/locales/fr/gcweb.json

```
@@ -122,5 +122,8 @@
       "label": "Année",
       "placeholder": "AAAA"
     }
+  },
+  "screen-reader": {
+    "newtab": "S'ouvre dans un nouvel onglet"
```

Should be in lowercase

```suggestion
    "newtab": "s'ouvre dans un nouvel onglet"
```